### PR TITLE
feat: Show mod version in LocalMods View

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Added based on user feedback and tbh it also makes sense to show it ^^

Rendered:
image

Requires:
- [x] #158

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
PR is technically done but leaving as draft until #158 is merged cause it'll simplify the diff a lot for review ^^

**Comment by GeckoEidechse** [comment]
CI is failing cause Ubuntu Azure repos are dead (again)

File: src-vue/src/views/mods/LocalModsView.vue [hunk 1]
```diff
@@ -14,6 +14,7 @@
                     </template>
                 </el-popconfirm>
                 {{ mod.name }}
+                (v{{ mod.version }})
```
Comment (Alystrasz) [inline src-vue/src/views/mods/LocalModsView.vue:17]: Given you've defined version type as `Option<String>`, what happens here if `mod.version` is null?

Comment (GeckoEidechse) [reply src-vue/src/views/mods/LocalModsView.vue:17]: Just turns into nothing it seems.
image

Given that it is such a rare occasion should we add edge case handling (I think Northstar prints `0.0.0` as the version number then) or just ignore?

Comment (Alystrasz) [reply src-vue/src/views/mods/LocalModsView.vue:17]: If this happens, you shouldn't display version indicator at all.

Comment (GeckoEidechse) [reply src-vue/src/views/mods/LocalModsView.vue]: Aight, done with 5274aa9ada22d9299e87fb41c4d42214517a97fd

**Review by Alystrasz — APPROVED** [review]
LGTM, confirmed working on Linux.

**Pull request merged**